helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml

```diff
@@ -13,20 +13,20 @@ metadata:
     nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.ingress.allowed | quote }}
 spec:
   tls:
-  {{- range .Values.ingress.hosts }}
+   {{- range .Values.ingress.hosts }}
   {{- $certSecret := .cert_secret -}}
   {{- $domainSuffix := .suffix }}
   - hosts:
     {{- range $.Values.ingress.prefixes }}
-      - {{ . }}{{ $domainSuffix }}
+      - {{ . }}-prisoner-content-hub{{ if $.Values.ingress.qualifier }}{{$.Values.ingress.qualifier}}{{- end}}{{ $domainSuffix }}
```

Comment:
this value is repeated on line 29. It is fairly complex, can we use a variable in this context?

Reply:
not really as each one is scoped to its own loop